pkg/common/scheduler/statefulset/scheduler.go

```diff
@@ -37,14 +42,28 @@ import (
 	"knative.dev/eventing-kafka/pkg/common/scheduler"
 )
 
+type SchedulerPolicyType int
+
+const (
+	// MaxFillup policy type adds replicas to existing pods to fill them up before adding to new pods
+	MaxFillup SchedulerPolicyType = iota
+	// EvenSpread policy type spreads replicas uniformly across zones in different regions for HA, and within each zone, fills up replicas in existing and new pods
```

Comment:
I would generalize this statement by saying vreplicas are spread across [failure-domains such as regions, zones, nodes](https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/).